CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ project(safestring)
 option(BUILD_UNITTESTS "Build also project unit-tests" OFF)
 
 if (NOT DEFINED BUILD_OPT_DEFAULT)
-  set (BUILD_OPT_DEFAULT YES CACHE PATH "Build option default")
+  set (BUILD_OPT_DEFAULT YES CACHE BOOL "Build option default")
```

Comment:
Would it not be better to change this to an `option()`?

Reply:
Yes, I think it would be better to change it into an option. I went for the type change because I wasn't sure if there was a reason why it wasn't already an option.